e2e/tests.spec.ts

```diff
@@ -0,0 +1,56 @@
+import {test, expect} from '@playwright/test'
+import {writeInMonaco} from './fixtures'
+
+import * as zod from '../src/zod'
+
+test('has title "Zod Playground', async ({page}) => {
+  await page.goto('/')
+
+  await expect(page).toHaveTitle(/Zod Playground/)
+})
+
+test('has header with title, share, theme toggler and github repo link', async ({page}) => {
+  await page.goto('/')
+
+  await expect(page.getByText('Zod Playground')).toBeVisible()
+  await expect(page.getByRole('button', {name: 'Share'})).toBeVisible()
+  await expect(page.getByLabel('Toggle color scheme')).toBeVisible()
+  await expect(page.getByRole('banner').getByRole('link')).toHaveAttribute(
+    'href',
+    'https://github.com/marilari88/zod-playground',
+  )
+})
+
+test('zod version switch', async ({page}) => {
+  await page.goto('/')
+
+  const latestZodVersion = (await zod.getVersions('latest'))[0]
+  const anotherZodVersion = (await zod.getVersions()).find(
+    (zVersion) => zVersion !== latestZodVersion,
+  )
```

Comment:
So, if you're not much used to e2e testing this might seem weird but writing it like this, makes the test more robust for a couple of reasons:

1. Now we're also testing the getVersions function (the more, the better).
2. One might say “what happens when the external service provided by jsdelivr.com stops working?” easy, the tests fail as they should, because jsdeliver is a dependency needed by the app to work properly, and therefore this failing test would signal that the app itself is not working as intended.